wordpress-admin/wordpress-admin.scss

```diff
@@ -0,0 +1,192 @@
+$swal2-actions-justify-content: flex-end;
+$swal2-actions-margin: 8px auto 0;
+
+// PROGRESS STEPS
+$swal2-progress-step-background: white;
+$swal2-progress-step-color: #333;
+$swal2-active-step-background: $wordpress-blue;
+
+// ICONS
+$swal2-icon-animations: false;
+$swal2-icon-margin: -36px -4px 0 -24px;
+$swal2-icon-zoom: 0.7;
+$swal2-success: $wordpress-green;
+$swal2-error: $wordpress-red;
+$swal2-warning: $wordpress-yellow;
+$swal2-info: $wordpress-blue;
+$swal2-question: $wordpress-blue;
+
+// FOOTER
+$swal2-footer-border-color: #555;
+$swal2-footer-color: darken($swal2-content-color, 15%);
+
+// INPUT
+$swal2-input-height: 30px;
+$swal2-input-padding: 0 8px;
+$swal2-input-border: 1px solid #7e8993;
+$swal2-input-border-radius: 4px;
+$swal2-input-box-shadow: 0 0 0 transparent;
+$swal2-input-focus-border: 1px solid $wordpress-blue;
+$swal2-input-focus-outline: 2px solid transparent;
+$swal2-input-focus-box-shadow: 0 0 0 1px $wordpress-blue;
+$swal2-input-font-size: 14px;
+$swal2-input-background: lighten($swal2-dark-theme-black, 5%);
+$swal2-input-color: $swal2-dark-theme-white;
+$swal2-input-transition: none;
+
+// VALIDATION MESSAGE
+$swal2-validation-message-justify-content: flex-start;
+$swal2-validation-message-padding: 4px 13px;
+$swal2-validation-message-background: transparent;
+$swal2-validation-message-color: $swal2-dark-theme-white;
+$swal2-validation-message-icon-background: $wordpress-red;
+$swal2-validation-message-icon-color: $swal2-white !default;
+$swal2-validation-message-icon-zoom: 0.7;
+
+// COMMON VARIABLES FOR CONFIRM AND CANCEL BUTTONS
+$swal2-button-darken-hover: transparent;
+$swal2-button-darken-active: transparent;
+$swal2-button-focus-box-shadow: 0 0 1px 1px;
+
+// CONFIRM BUTTON
+$swal2-confirm-button-background-color: $wordpress-blue;
+$swal2-confirm-button-border: 0;
+$swal2-confirm-button-border-radius: 3px;
+$swal2-confirm-button-color: $swal2-dark-theme-white;
+$swal2-confirm-button-font-size: 13px;
+
+// CANCEL BUTTON
+$swal2-cancel-button-font-size: 13px;
+
+// TOAST
+$swal2-toast-show-animation: fadeInUpBig .65s;
+$swal2-toast-hide-animation: fadeOut 0.5s;
+$swal2-toast-background: $swal2-dark-theme-black;
+$swal2-toast-box-shadow: 0 2px 4px rgba(0, 0, 0, 0.3);
+$swal2-toast-close-button-width: auto;
+$swal2-toast-close-button-height: auto;
+$swal2-toast-close-button-line-height: 16px;
+$swal2-toast-padding: 16px 24px;
+$swal2-toast-title-font-size: 13px;
+$swal2-toast-content-font-size: 13px;
+$swal2-toast-input-font-size: 13px;
+$swal2-toast-validation-font-size: 13px;
+$swal2-toast-buttons-font-size: 13px;
+$swal2-toast-footer-font-size: 13px;
+
+// TIMER POGRESS BAR
+$swal2-timer-progress-bar-height: .25em;
+$swal2-timer-progress-bar-background: $wordpress-blue;
+
+@import '~sweetalert2/src/sweetalert2';
+
+// Overrides
+.swal2-container {
+  padding: 24px 16px;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
```

Comment:
we prefer not setting `font-family` for popups, as they should be aligned with an app, e.g. inherit `font-family`. I think the app `font-family` for Wordpress Admin is `-apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;` If so, this line isn't needed.

Reply:
@limonte That's right that SweetAlert will inherit the font family while in the WP dashboard but if someone wants the WP style toasts in a non-WP application then it wont look like WordPress anymore unless they set the font. 

For example,

<img width="384" alt="Screen Shot 2019-12-10 at 11 44 18 PM" src="https://user-images.githubusercontent.com/1478421/70549943-9914ff80-1ba7-11ea-8a11-a8ab4eb7a49e.png">

It looks fine to me, but it's no longer WordPress style. 

I'll remove it and add a note in the readme, but let me know if you want something else.